PRODUCTIONISATION.md

```diff
@@ -49,8 +49,8 @@ Ensuring that your project adheres to the following RFCs is a good first step as
 ### 3. Environments
 
 * 3.1 Application has corresponding environments and hosting for:
-  * [ ] 3.1.1 `testing` for internal delivery team
-  * [ ] 3.1.2 `staging` for showcasing to customer
+  * [ ] 3.1.1 `testing` for internal delivery team, this environment should only be accessible to engineers
+  * [ ] 3.1.2 `staging` for showcasing to customer, with restricted access and its setup should mirror production
```

Comment:
>restricted access

There isn't a real benefit to protecting staging like this as:

- there isn't real data
- anything in staging gets quickly promoted to production anyway (so IP leakage doesn't really apply here)
- it contradicts the next statement of "setup should mirror production".

Reply:
See https://github.com/madetech/productionisation/pull/51/commits/323f30c34e0ebac807361540bbf598dcc86c3e66 for changes.